common-money.lic

```diff
@@ -40,26 +40,45 @@ module DRCM
     check_wealth(hometown_currency(hometown))
   end
 
-  def ensure_copper_on_hand(copper, hometown)
+  def ensure_copper_on_hand(copper, hometown, bankbot_enabled, bankbot_name, bankbot_room_id)
```

Comment:
What if instead of passing these four arguments, you had

`def ensure_copper_on_hand(copper, settings)`

Then we'd still have the advantage of having local settings, but not every caller would have to know about the specific settings involved.